pyproject.toml

```diff
@@ -27,13 +27,17 @@ dependencies = [
   "structlog==23.1.0",
   "httpx==0.24.0",
   "python-dotenv==1.0.0",
+  "typer==0.9.0",
```

Comment:
for some reason the lint step in the CI is complaining that it could not import typer

Reply:
i know. I guess its somehow related to hatch and caching.